### PR TITLE
Make sure <unpack>/image/etc gets into the system-data directory

### PR DESCRIPTION
**Pull request by warsaw**
Closes LP: #1632134 and replaces #69.  @mvo5

### CODE REVIEW

File: ubuntu_image/builder.py [hunk 1]
```diff
@@ -117,7 +117,12 @@ def load_gadget_yaml(self):
     def populate_rootfs_contents(self):
         src = os.path.join(self.unpackdir, 'image')
         dst = os.path.join(self.rootfs, 'system-data')
-        shutil.move(os.path.join(src, 'var'), os.path.join(dst, 'var'))
+        for subdir in os.listdir(src):
+            # LP: #1632134 - copy everything under the image directory except
+            # /boot and /home.  For now, we even copy files.
+            if subdir not in ('boot', 'home'):
```
Comment (vorlonofportland) [inline ubuntu_image/builder.py]: +1 for the /boot exclusion. Why exclude /home?

Comment (warsaw) [inline ubuntu_image/builder.py]: That's a question for @mvo5 who suggested this in #69

Comment (warsaw) [inline ubuntu_image/builder.py]: FWIW, there doesn't actually seem to be a /home in the pc or pi2 gadgets.

Comment (warsaw) [inline ubuntu_image/builder.py]: I've thought about this more and in the absence of a motivating use case, I will not exclude /home.

**Pull request merged**